src/app/newSettings/components/BillingOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import { useSelector } from 'react-redux';
+import { t } from 'i18next';
+
+import { RootState } from 'app/store';
+import { PlanState } from 'app/store/slices/plan';
+import { getSubscriptionData, getNextBillingDate } from '../utils/suscriptionUtils';
+import localStorageService from 'app/core/services/local-storage.service';
+import { bytesToString } from '../../drive/services/size.service';
+
+import Card from 'app/shared/components/Card';
+
+const BillingOverview = () => {
+  const plan = useSelector<RootState, PlanState>((state) => state.plan);
```

Comment:
Avoid attaching components to redux states, instead of adding it here, pass it through props.
Same with other props, if it something that is not related directly with the representation of a value in the component, try to pass it through props. 
Let's try to think as if the new components are going to go into the new component library, so it will be easier to separate what has to go into the props and what can go into the component.

Reply:
This is already being used in other components, such as Overview, I understand that this will change when the new api calls are implemented and it is better to use this for now rather than mock data.

Reply:
Maybe in a `container` or `view`, if so, please, move it out of the `components` folder to its corresponding directory.

Reply:
Finally I have moved this logic to the view